src/__test__/home.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import store from '../redux/configureStore';
+import Home from '../components/home/home';
+
+describe('Home Test', () => {
+  test('display pairs', () => {
+    const home = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Home />
+        </BrowserRouter>
+      </Provider>,
+    );
+    expect(home).toMatchSnapshot();
+  });
```

Comment:
- [ ] Kindly add more tests for the rest of the components of your app 
- [ ] Also create integration tests for your application using the React Testing Library.
   - You may need to mock the access to the API, so that your tests don't send actual requests.
   - You may need to mock the connection to the Redux Store.